scripts/chain_configs/localterra.json

```diff
@@ -180,79 +180,6 @@
     "label": "Astroport Oracle"
   },
   "createPairs": {
-    "pairs": [
```

Comment:
Maybe let's leave some template to easy fill fields  to deploy pairs?

Reply:
Maybe, but if you don't need the template you have to remove it